internal/common/common.go

```diff
@@ -1,15 +1,44 @@
 package common
 
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
 const (
-	// LabelKeyNumaplaneInstance Resource metadata labels (keys and values) used for tracking
-	LabelKeyNumaplaneInstance = "numaplane.numaproj.io/tracking-id"
 	// SSAManager is the default numaplane manager name used by server-side apply syncs
 	SSAManager = "numaplane-controller"
 	// EnvLogLevel log level that is defined by `--loglevel` option
 	EnvLogLevel = "NUMAPLANE_LOG_LEVEL"
 
+	NumaflowAPIGroup = "numaflow.numaproj.io"
+
+	NumaflowAPIVersion = "v1alpha1"
+
+	// LABELS:
+
+	// LabelKeyNumaplaneInstance Resource metadata labels (keys and values) used for tracking
+	LabelKeyNumaplaneInstance = "numaplane.numaproj.io/tracking-id"
+
 	// LabelKeyNumaplaneControllerConfig is the label key used to identify the configmap for the controller definitions
 	LabelKeyNumaplaneControllerConfig = "numaplane.numaproj.io/config"
 	// LabelValueNumaplaneControllerConfig is the label value used to identify the configmap for the controller definitions
 	LabelValueNumaplaneControllerConfig = "numaflow-controller-definitions"
+
+	LabelKeyISBServiceNameForStatefulSet = "numaflow.numaproj.io/isbsvc-name"
+
+	LabelKeyISBServiceNameForPipeline = "numaplane.numaproj.io/isbsvc-name"
+)
+
+var (
+	NumaflowGroupVersion = fmt.Sprintf("%s/%s", NumaflowAPIGroup, NumaflowAPIVersion)
+
+	PipelineGVR = schema.GroupVersionResource{
+		Group:    NumaflowAPIGroup,
+		Version:  NumaflowAPIVersion,
+		Resource: "pipelines",
+	}
+
+	DataLossPrevention bool
```

Comment:
So, currently I'm only ever using the value of this once at the beginning. The users all just create a local variable equal to this value on start up. I originally had it so that it could theoretically get changed in the middle of running. But then I was thinking that that would mean having to potentially write code to address it being changed from true to false or false to true, which could create more complexities. Do you think maybe it's worth a thought exercise to figure out just how complicated or not complicated it would be?

One thing to note is that if we do modify it and don't modify the Numaplane Controller Deployment itself, we would have to restart the Pods on every cluster. This can be done by writing a fairly simple script, however, assuming that we have the right to write to resources in numaplane-system namespace. (We could pursue getting myself and others "ClusterAppAdmin" privilege to our namespace)

Reply:
Update: per [this](https://intuit-teams.slack.com/archives/C05K82P3N9J/p1722295803818349) conversation, it seems we should have this

Reply:
I think we would want this to be configurable dynamically. We can track this as a separate issue and do this as a follow up though.

Reply:
okay, added Issue for it: https://github.com/numaproj/numaplane/issues/155